scripts/sem_bleaching_map.py

```diff
@@ -1,42 +1,47 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-'''
+"""
 Created on 20 Mar 2014
 
-@author: Éric Piel
+@author: Éric Piel, modified by Aaro Vakevainen, modified by Lennard Voortman
 
 This is a script to scan a region with a e-beam and observe the reduction
 of fluorescence on an optical image for each point scanned (due to bleaching).
 
 run as:
-./scripts/sem_bleaching_map.py --dt=0.01 --roi=0.1,0.1,0.2,0.2 --pxs=7800 --subpx=49 --output filename.h5
+./scripts/sem_bleaching_map.py --dt=600e-6 --dtsem=30e-6, --roi=0.4,0.4,0.95,0.95 --pxs=376 --subpx=6 --lpower=0.3 --lemissions=1 --exptime=1 --output=filename.tiff -olp 0.3 -ole 1 -oet 1 --gridpitch=4 --drift=3 --anchor=0.01,0.01,0.1,0.1
 
---dt defines the dwell time when scanning (time required to bleach one spot/sub-pixel)
+--dt defines the dwell time in seconds when scanning (time required to bleach one spot/sub-pixel)
+--dtsem is the required time in seconds to take a SEM image
 --roi the top-left and bottom-right points of the region to scan (relative to
       the SEM field of view).
---pxs the distance between the centers of two consecutive spots, in nm. It 
+--pxs the distance between the centers of two consecutive spots, in nm. It
       should be compatible with the current SEM magnification.
---subpx the number of subpixels (must be a square of a integer).
+--subpx the number of subpixels (must be a square of an integer).
       For each pixel acquired by the CCD, each subpixel is scanned by the ebeam.
---output indicates the name of the file which will contain all the output. It 
+--output indicates the name of the file which will contain all the output. It
          should finish by .h5 (for HDF5) or .tiff (for TIFF).
---lpower defines the excitation light power on the first fluorescence pixel. The value can be 
-		 between 0.004 and 0.4 (in W). Power will be gradually scaled up to the maximum (0.4 W) 
-         on the last pixel to compensate photobleaching effect in the signal drop.
+--lpower defines the excitation light power in Watt
+--lemissions gives the emission source (numbered from 0 to n-1).
+--exptime gives the exposure time in seconds
+-olp is the excitation light power used for overview image
+-ole is the emission source (numbered from 0 to n-1) used for overview image
+-oet is the exposure time that is used for overview image
```

Comment:
I believe these 3 parameters should be lists not a singular value. Can you make sure of this? They are handled in the code as lists..